ApiUsage/ArmBased/Program.cs

```diff
@@ -27,76 +28,106 @@ public static async Task Main(string[] args)
+        {
             var content = new MultipartFormDataContent();
             Console.WriteLine("Uploading...");
-            // Get the video from URL
-
-            // As an alternative to specifying video URL, you can upload a file.
-            // Remove the videoUrl parameter from the query params below and add the following lines:
-            // FileStream video =File.OpenRead(Globals.VIDEOFILE_PATH);
-            // byte[] buffer =new byte[video.Length];
-            // video.Read(buffer, 0, buffer.Length);
-            // content.Add(new ByteArrayContent(buffer));
 
-            var queryParams = CreateQueryString(
+            try
+            {
+                // Get the video from URL
+                var queryParams = CreateQueryString(
                 new Dictionary<string, string>()
                 {
-            {"accessToken", accessToken},
-            {"name", "video sample"},
-            {"description", "video_description"},
-            {"privacy", "private"},
-            {"partition", "partition"},
-            {"videoUrl", VideoUrl},
+                    {"accessToken", acountAccessToken},
+                    {"name", "video sample"},
+                    {"description", "video_description"},
+                    {"privacy", "private"},
+                    {"partition", "partition"},
+                    {"videoUrl", VideoUrl},
                 });
-            var uploadRequestResult = await client.PostAsync($"{apiUrl}/{accountLocation}/Accounts/{accountId}/Videos?{queryParams}", content);
-            var uploadResult = await uploadRequestResult.Content.ReadAsStringAsync();
 
-            // Get the video ID from the upload result
-            string videoId = JsonSerializer.Deserialize<Video>(uploadResult).Id;
-            Console.WriteLine("Uploaded");
-            Console.WriteLine("Video ID:");
-            Console.WriteLine(videoId);
+                // As an alternative to specifying video URL, you can upload a file.
+                // Remove the videoUrl parameter from the query params below and add the following lines:
+                // FileStream video =File.OpenRead(Globals.VIDEOFILE_PATH);
+                // byte[] buffer =new byte[video.Length];
+                // video.Read(buffer, 0, buffer.Length);
+                // content.Add(new ByteArrayContent(buffer));
+
+                // For more info on this API see API portal (https://api-portal.videoindexer.ai/api-details#api=Operations&operation=Upload-Video)
+                var uploadRequestResult = await client.PostAsync($"{apiUrl}/{accountLocation}/Accounts/{accountId}/Videos?{queryParams}", content);
+                VerifyStatus(uploadRequestResult);
+                var uploadResult = await uploadRequestResult.Content.ReadAsStringAsync();
+
+                // Get the video ID from the upload result
+                var videoId = JsonSerializer.Deserialize<Video>(uploadResult).Id;
+                Console.WriteLine("Uploaded");
```

Comment:
unify all those Console. to 1 statement where all the information exsits clearly .